Update README.md for clarity and additional information (#506) (#28)
amd/gaia: README.md

[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
[![GitHub issues](https://img.shields.io/github/issues/amd/gaia)](https://github.com/amd/gaia/issues)

> ⚠️ **IMPORTANT**: GAIA's Hybrid mode **only supports AMD Ryzen AI HX 300 series or newer processors**. For older AMD processors or non-AMD systems, please use the Generic installer. For more details, see [here](https://www.amd.com/en/products/software/ryzen-ai-software.html#tabs-2733982b05-item-7720bb7a69-tab).

Reviewer: ```suggestion
⚠️ **IMPORTANT**: GAIA's Hybrid mode **only supports AMD Ryzen AI 300 series or newer processors**. For older AMD processors or non-AMD systems, please use the Generic installer. For more details, see [here](https://www.amd.com/en/products/software/ryzen-ai-software.html#tabs-2733982b05-item-7720bb7a69-tab).
```

Should apply to all 300-series, not only HX 300-series.

The quickest way to get started with GAIA is by using one of the provided installers. There are two options available:

1. **GAIA_Hybrid_Installer.exe**: For running agents with the Hybrid (NPU+iGPU) execution on Ryzen AI PCs. This is the recommended installer that offers the best performance. **Requires AMD Ryzen AI 9 HX 300 series or newer processor**.

> 1. **GAIA_Installer.exe**: For running agents on non-Ryzen AI PCs, this uses Ollama as the backend.

Reviewer: ```suggestion
1. **GAIA_Installer.exe**: For running agents on non-Ryzen AI 300-series PCs, this uses Ollama as the backend.
```

Remember PHX/HWK are still Ryzen AI, but they should use the Ollama backend for now.

Each installer includes both a CLI tool and a GUI. The installation process typically takes about 5-10 minutes, depending on your Wi-Fi connection, and provides everything you need to start working with LLMs.

⚠️ **NOTE**: When running GAIA using the Hybrid mode, please make sure to disable any discrete third-party GPUs in Device Manager.

## Installation Steps

![image](./data/img/gaia-setup.png)

To install the GAIA application, please follow these steps:

> 1. Make sure you meet the minimum [system requirements](#system-requirements) - **For Hybrid mode, you must have AMD Ryzen AI 9 HX 300 or newer processor**

Reviewer: ```suggestion
1. Make sure you meet the minimum [system requirements](#system-requirements) - **For Hybrid mode, you must have an AMD Ryzen AI 300-series processor**
```

- Ollama *(Generic mode only)*
- All required Python packages

# Troubleshooting

If you encounter issues with GAIA, please check the following common solutions:

## Driver Issues

> - **NPU Driver Incompatibility**: For Hybrid mode, ensure you have NPU Driver `32.0.203.237` or `32.0.203.240`. Driver `32.0.203.242` may experience issues.

Reviewer: FYI there are a bunch of new drivers now. 250, 252, and 257 (the latter of which is the official driver for Ryzen AI SW 1.4).

Author: Will update comment once we transition to RAI 1.4.

Reviewer: The new drivers are in the wild already. For example, windows update may automatically put your user on to driver 250 (happened to me). So suggest noting the existence of that driver now to avoid confusion.
